### PR TITLE
chore: update the pricing link and plan features

### PR DESCRIPTION
**Pull request by ecmadao**

### CODE REVIEW

File: pages/pricing/index.vue [hunk 1]
```diff
@@ -499,7 +499,7 @@ export default defineComponent({
   methods: {
     onButtonClick(plan: Plan) {
       if (plan.type === PlanType.TEAM) {
-        window.open("https://hub.bytebase.com/pricing?plan=team", "__blank");
+        window.open("https://hub.bytebase.com/pricing?plan=team&source=official-website", "__blank");
```
Comment (tianzhou) [inline pages/pricing/index.vue:502]: Not sure if we need more granular sources like: main-site.pricing (should be consistent with our funnel naming)

Comment (ecmadao) [reply pages/pricing/index.vue:502]: Seems the "main-site.{page}" format in bytebase.com can show more detail about the source

Comment (ecmadao) [reply pages/pricing/index.vue:502]: We can add the segment event and pv track for pricing page in main site. Will update later.

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**